src/ucp/api/ucp.h

```diff
@@ -1299,6 +1310,61 @@ ucs_status_t ucp_ep_flush(ucp_ep_h ep);
  * by a user, then the user is responsible for segmentation and subsequent
  * management.
  *
+ * <table>
+ * <caption id="ucp_mem_map_matrix">Matrix of behavior</caption>
+ * <tr><th>Parameter/Flag                  <th colspan="8">Value
+ * <tr><td>@ref UCP_MEM_MAP_NONBLOCK       <td colspan="8" align="center">
+ *                                         0/1 - the value affects only
+ *                                         registration/mapping phase </td>
+ * <tr><td>@ref UCP_MEM_MAP_ALLOCATE       <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ * <tr><td>@ref UCP_MEM_MAP_FIXED          <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">1</td>
+ *                                         <td align="center">1</td>
+ * <tr><td>@ref ucp_mem_map_params.address <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">defined</td>
+ *                                         <td align="center">0</td>
+ *                                         <td align="center">defined</td>
+ *                                         <td align="center">defined</td>
+ *                                         <td align="center">defined</td>
+ * <tr><td>Result                          <td>@ref anch_err "error"
+ *                                         <td>@ref anch_alloc_reg "alloc+register"
+ *                                         <td>@ref anch_err "error"
+ *                                         <td>@ref anch_reg "register"
+ *                                         <td>@ref anch_err "error"
+ *                                         <td>@ref anch_alloc_hint_reg "alloc+register,hint"
+ *                                         <td>@ref anch_err "error"
+ *                                         <td>@ref anch_alloc_fixed_reg "alloc+register,fixed"
+ * </table>
```

Comment:
When converted to pdf, this table has some formatting and alignment issues.  Maybe using shorter names to print in the references to the parameter flags will help.  If it could look more like the ASCII matrix in ucp_mm.c, I think that would help for readability.